app/services/proceeding_type_populator.rb

```diff
@@ -0,0 +1,18 @@
+require 'csv'
+
+class ProceedingTypePopulator
+  def self.call
+    new.call
+  end
+
+  def file_path
+    Rails.root.join('db', 'seeds', 'proceeding_types.csv').freeze
```

Comment:
Pedantic, and certainly a matter of taste ... but I think this is easier to read:
```ruby
Rails.root.join('db/seeds/proceeding_types.csv').freeze
```